### PR TITLE
Add NFTs page to the wallet 

### LINKED ISSUE #2595: Add NFTs page to the wallet
- [ ] create new feature flag and add new changes based on the flag
- [ ] add a new tab to the bottom ribbon
image

- [ ] remove swaps page from bottom ribbon, move it to the Wallet page (switch between new and old versions based on the flag)
image

- [ ] fix the order of pages in the bottom ribbon

### PR DESCRIPTION
**Pull request by kkosiorowska**
Closes #2595 

This PR adds an NFTs page to the wallet.

Changes:
- add a new NFTs tab to the bottom ribbon
- move the swap page to the Wallet page

**UI**

Before
Screenshot 2022-11-18 at 19 05 19

After
Screenshot 2022-11-18 at 19 04 23

Screenshot 2022-11-18 at 19 30 33

**To test**
Set up `SUPPORT_NFT_TAB` to `true`.
- [ ] check that the NFTs tab is added to the bottom ribbon
- [ ]  check if the swap tab is removed from the bottom ribbon
- [ ] check that the swap page is moved to the Wallet page

Latest build: [extension-builds-2640](https://github.com/tallycash/extension/suites/9462540285/artifacts/448001089) (as of Wed, 23 Nov 2022 09:03:25 GMT).

### CODE REVIEW

File: .env.defaults [hunk 1]
```diff
@@ -25,6 +25,7 @@ SUPPORT_FORGOT_PASSWORD=false
 ENABLE_ACHIEVEMENTS_TAB=true
 SUPPORT_ACHIEVEMENTS_BANNER=false
 SWITCH_RUNTIME_FLAGS=false
+HIDE_NFTS=true
```
Comment (mhluongo) [inline .env.defaults:28]: We should prefer "SHOW_NFTS=false" as the default, or alternatively something like "SUPPORT_NFT_TAB".

File: ui/components/Wallet/WalletAccountBalanceControl.tsx [hunk 2]
```diff
@@ -65,25 +68,38 @@ export default function WalletAccountBalanceControl(
...
-                  <SharedButton
-                    onClick={handleClick}
-                    iconSmall="receive"
-                    size="medium"
-                    type="tertiary"
-                    iconPosition="left"
-                  >
-                    {t("wallet.receive")}
-                  </SharedButton>
+                <div className="actions_button_wrap">
```
Comment (greg-nagy) [inline ui/components/Wallet/WalletAccountBalanceControl.tsx:71]: This should also depend on the feature flag.

**Review by jagodarybacka — APPROVED** [review]
Non-blocking comment, I think it is fine :)

Will wait for @VladUXUI to take a look too

**Review by jagodarybacka — APPROVED** [review]
✅

**Review by greg-nagy — CHANGES_REQUESTED** [review]
This looks good, and works well! 

Nice job here 🚀

After blockers are fixed this is good to merge.

File: ui/components/Shared/SharedSquareButton.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    onClick?.(event)
+    if (linkTo) {
+      setNavigateTo(linkTo)
+    }
+  }
+
+  if (navigateTo && navigateTo === linkTo) {
+    return <Redirect push to={linkTo} />
+  }
```
Comment (hyphenized) [inline ui/components/Shared/SharedSquareButton.tsx:51]: We can let the parent component handle navigation through the `onClick` prop

e.g.
```tsx
function BiggerComponent(){
  ...
  const history = useHistory();
  ...
  return <...>
    <SharedSquareButton onClick={()=> history.push("/somewhere") }>somewhere</SharedSquareButton>
  </...>
}

File: ui/components/Wallet/WalletAccountBalanceControl.tsx [hunk 4]
```diff
@@ -65,25 +68,38 @@ export default function WalletAccountBalanceControl(
...
+                  <div className="button_wrap">
```
Comment (hyphenized) [inline ui/components/Wallet/WalletAccountBalanceControl.tsx:72]: I think these buttons should only change if the feature flag is enabled

**Review by hyphenized — CHANGES_REQUESTED** [review]
Great work! Left some comments.

**Comment by mhluongo** [comment]
@VladUXUI @mr-michael this is the sort of change that could grow or shrink swap volume significantly. Wanted to make sure you both were sure that's where we want the swap page, and that we're keeping track of rough per-install swap volume so we can revert if it's negative.

**Comment by VladUXUI** [comment]
> Non-blocking comment, I think it is fine :)
> 
> Will wait for @VladUXUI to take a look too

Looking good! yes the button on the asset cards can remain

**Comment by VladUXUI** [comment]
> @VladUXUI @mr-michael this is the sort of change that could grow or shrink swap volume significantly. Wanted to make sure you both were sure that's where we want the swap page, and that we're keeping track of rough per-install swap volume so we can revert if it's negative.

Swaps had a very different account/network context than the rest of the pages, so imo had to be moved. The placement is one that users are very familiar with as this is a pattern employed by most extensions or application wallets.
I don't think it's more difficult to find there then it would be in our main menu. We also have swaps button in more places than your usual extension/app.
Fairly confident it won't negatively affect our swaps volume, but good point about tracking! Will do that and see if we need to revert

**Review by VladUXUI — APPROVED** [review]

File: ui/components/Shared/SharedSquareButton.tsx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+  return (
+    <button type="button" aria-label={ariaLabel} onClick={handleClick}>
+      <div className="content_wrap">
```
[the remark this answers is not in the document]
Comment (kkosiorowska) [reply ui/components/Shared/SharedSquareButton.tsx:55]: Let's make it simple.

**Review by greg-nagy — APPROVED** [review]
looks good, let's 🚢

**Pull request merged**